src/Worker.php

```diff
@@ -80,9 +80,11 @@ private function tick($queue)
         }
 
         try {
+            $this->event->acknowledge($message);
```

Comment:
Not sure there's much value in indicating that we got the message but haven't done anything with it yet unless that provides a way to short-circuit the process of executing the job.